metpy/gridding/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2008-2015 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from .gridding_functions import *  # noqa
+from .points import *  # noqa
+from .triangles import *  # noqa
+from .polygons import *  # noqa
+from .interpolation import *  # noqa
+
+__all__ = []
+__all__.extend(gridding_functions.__all__)  # pylint: disable=undefined-variable
+__all__.extend(points.__all__)  # pylint: disable=undefined-variable
+__all__.extend(triangles.__all__)  # pylint: disable=undefined-variable
```

Comment:
Do you really think we should be including everything in `points`, `triangles`, and `polygons` as main API functions? That means eventually we intend to have everything that's been `exporter.export`-ed as API's that will not change and are supported to use. Certainly I see some functions in there (like `dist_2`) that definitely aren't, and I'm not sure any of these are things I consider beyond internal details from a "Meteorology for Python" perspective.